src/state/epics.js

```diff
@@ -1,11 +1,12 @@
 import { combineEpics } from 'redux-observable';
-import { mapTo } from 'rxjs/operators';
+import { FetchBeer } from './entities/beer/operations/FetchBeer';
```

Comment:
Could consider exporting epics from an index file to be used here. 
It would look something like the following:

// index.js in operations folder
combineEpics(foo,bar,baz,...)

// epic.js 
import BeerEpics from './path/to/operations/index'
combineEpics(BeerEpics, Entity2Epics, EntityXEpics,...)

Would allow for more scalable code. Might be overkill for this. Something to consider I guess.